src/parse.rs

```diff
@@ -1319,31 +1311,12 @@ impl<'a> FirstPass<'a> {
                 }
                 '\\' => {
```

Comment:
I think we can skip this branch altogether now, no?

Reply:
Pretty sure not, you have to resolve the difference between `(title\))` and `(title))`. The former is a valid link title, the second isn't. As you say, tests will help settle the question, I'll add those.